compiler-core/src/build.rs

```diff
@@ -184,6 +185,7 @@ pub enum Mode {
     Dev,
     Prod,
     Lsp,
+    PackageInterface,
```

Comment:
Package interface is not a compiler mode, this must be removed.

Reply:
Okay, removed that!